src/covertool.erl

```diff
@@ -83,8 +83,7 @@ generate_report(Config, Modules) ->
     Prolog = ["<?xml version=\"1.0\" encoding=\"utf-8\"?>\n",
               "<!DOCTYPE coverage SYSTEM \"http://cobertura.sourceforge.net/xml/coverage-04.dtd\">\n"],
 
-    {MegaSecs, Secs, MicroSecs} = erlang:timestamp(),
```

Comment:
That is fine with me too; it appears to also be compatible with both R16 and 18.0.